src/components/attendee/ChatContainer.module.scss

```diff
@@ -100,11 +100,15 @@
   margin-bottom: space.empty(3);
 
   @media (min-width: breakpoint.$md) {
-    min-height: 90vh;
+    min-height: calc(100vh - 69px);
```

Comment:
[NB] Can we use a constant for this 69px value? Is it the height of the footer or something else?

Reply:
added more meaning to the calculations here with the comment: https://github.com/sparkletown/sparkle/pull/3038/commits/ac1c1c2c500155a189e76ade05363d5799e129be